src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -786,7 +786,7 @@ private function deleteAuditLogs(Document $project, callable $getProjectDB, stri
         $dbForProject = $getProjectDB($project);
 
         try {
-            $this->deleteByGroup(Audit::COLLECTION, [
+            $this->deleteByGroup(SQL::COLLECTION, [
```

Comment:
Why do we need this constant? can we abstract it behind the delete method? we already Iintialize the adapter so we should be able to know this internally.